packages/react-common/test/moratorium-banner.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { CovidMoratoriumBanner } from "../src";
+
+test("Moratorium Banner renders English text by default", () => {
+  const container = document.createElement("div");
+  document.body.appendChild(container);
+  try {
+    ReactDOM.render(<CovidMoratoriumBanner />, container);
+    expect(container.innerHTML).toContain("COVID-19 Update:");
+  } finally {
+    ReactDOM.unmountComponentAtNode(container);
+    document.body.removeChild(container);
+  }
```

Comment:
Cool, thanks for adding tests!  Would you be interested in doing these using [React Testing Library](https://testing-library.com/docs/react-testing-library/intro)?  It's what we use for tenants2 and generally makes tests a lot less verbose, among other things--for example, you'd be able to replace all of the above with something like this:

```jsx
const el = render(<CovidMoratoriumBanner/>);
el.getByText(/COVID-19 Update/i);
```

Up to you though, we could also just file an issue to do it later...  If you do take this on, make sure you add the testing library as a dev dependency using `yarn add --dev` (I think that's the flag at least).

Reply:
Oh interesting! Will file an issue to do this later!